docs/v1/README.md

```diff
@@ -5,62 +5,69 @@ specification document.
 
 # Markup version
 
-The markup version uses Madoko (https://www.madoko.net) to produce
+The markup version uses AsciiDoc (https://docs.asciidoctor.org/) to produce
 HTML and PDF versions of the documentation. Pre-built versions of the
 documentation are available on the [P4.org specifications
 page](https://p4.org/specs).
 
 
 Files:
-- `P4Runtime-spec.mdk` is the main file. 
-- assets: Figures
-  - `*.odg` - OfficeLibre source drawing file used to export images. These are
+- `P4Runtime-Spec.adoc` is the main file. 
+- resources: 
+  - figs
+    - `*.odg` - OfficeLibre source drawing file used to export images. These are
     bulk-rendered at build time into .svg and .png images via `soffice`
     command-line (required in build environment)
+  - fonts 
+      - `*.ttf` - Type font source file used to export fonts.
+  - theme: 
+      - `*.yaml` - Describes how PDF P4Runtime specification will be displayed.
+      - `*.css`  - Describes how HTML P4Runtime specification will displayed.
+      - `*.bib`  - Bibliography file that contains a list of bibliographical item, such as articles, books, and theses.
 - `Makefile` builds documentation in the build subdirectory
-- `p4.json` is providing custom syntax highlighting for P4. It is a rip off from
-  the cpp.json provided by Madoko (the "extend" clause does not work, my version
-  of Madoko asks for a "tokenizer" to be defined). Style customization for each
-  token can be done using CSS style attributes (see `token.keyword` in
-  `P4Runtime-spec.mdk`).
 
-## Building
+## Document Figures
 
-The easiest way to render the Madoko specification documentation is to use the
-`p4lang/p4rt-madoko:latest` Docker` image:
+Each image in the specification has a corresponding `.odg` file under
+`resources/figs/`. These are LibreOffice drawing files. The files are rendered into
+`.svg` and `.png` images (for HTML and PDF output, resepectively) at build time,
```

Comment:
Since the .svg and .png are rendered at build-time, why are you checking them into the repo?

Reply:
+1

Reply:
@smolkaj They are no longer being checked into the repo.  Davide removed them from this PR due to Chris's comment above.